src/main/java/org/gridsuite/filter/server/dto/expertfilter/expertrule/StringExpertRule.java

```diff
@@ -46,12 +64,9 @@ public boolean evaluateRule(Identifiable<?> identifiable) {
             case BEGINS_WITH -> StringUtils.startsWithIgnoreCase(identifiableValue, this.getValue());
             case ENDS_WITH -> StringUtils.endsWithIgnoreCase(identifiableValue, this.getValue());
             case EXISTS -> !StringUtils.isEmpty(identifiableValue);
+            case IN -> this.getValues().contains(identifiableValue);
+            case NOT_IN -> !this.getValues().contains(identifiableValue);
```

Comment:
Potential NPE

Reply:
Idem! Should manage at DTO level by a validator to do a cross validation between two fields. This will be corrected in another PR